app/src/main/java/com/strv/movies/ui/moviedetail/MovieDetailScreen.kt

```diff
@@ -82,29 +84,49 @@ fun MovieTrailerPlayer(videoId: String, progressSeconds: MutableState<Float>) {
 
 @Composable
 fun MoviePoster(movie: MovieDetail) {
-    AsyncImage(
-        model = "https://image.tmdb.org/t/p/w500${movie.posterPath}",
-        contentDescription = stringResource(id = R.string.movie_image),
-        modifier = Modifier
-            .padding(top = 16.dp)
-            .size(120.dp)
-    )
+    Column {
+
+        AsyncImage(
+            model = "https://image.tmdb.org/t/p/w500${movie.posterPath}",
+            contentDescription = stringResource(id = R.string.movie_image),
+            modifier = Modifier
+                .padding(top = 16.dp)
+                .size(120.dp),
+        )
+        Row {
```

Comment:
You would need `Row` if you want to place multiple composables in a row, but here you have only one `Text` so the `Row` seems superfluous to me. You can remove the `Row` and place the `Text` directly into `Column`.

Reply:
yes, makes sense, done